facturacion_mexico/cfdi_recibidos/tests/test_expense_account_resolver.py

```
@@ -0,0 +1,502 @@
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _table_exists(doctype: str) -> bool:
+	return frappe.db.table_exists(doctype)
+
+
+def _mapeo_sat_migrated() -> bool:
+	try:
+		return frappe.db.table_exists("Mapeo Equivalencias SAT")
+	except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)